src/pages/Lending/hooks/useRepaymentLockData.tsx

```diff
@@ -101,14 +118,14 @@ export const useRepaymentLockData = ({ accountId, assetId }: UseLendingPositionD
       const repaymentBlock = bnOrZero(last_open_height).plus(repaymentMaturity)
 
       const repaymentLock = bnOrZero(repaymentBlock)
-        .minus(blockHeight)
+        .minus(blockHeight!) // actually defined at runtime, see isRepaymentLockQueryEnabled
```

Comment:
the state of these variables is really hard to track in these query functions... wondering if we just handle invalid cases in case things change. to me this is only valid due to a combination of `isRepaymentLockQueryEnabled` and the fact that we are returning early on `if (!position)` which is only enabled if `!!accountId` && !!assetId`, which would imply we wouldn't skip L87 being the only check on `blockHeight`.

Just feels like some convoluted variable state that could easily change on accident that we may just want to account for to save ourselves in the future.

Reply:
Discussed this one out, agreed to avoid non-null assertions, even though things *should* not be null as per the enabled flow, safety first

https://github.com/shapeshift/web/pull/5721/commits/4b4ef66cfe12e30938961c73ea04c92f38c8b3de